### ISSUE TITLE
Feature Request - Add Adult Content on/off to preferences 

### ISSUE THREAD

**Issue by Pyxie** [piplongrun/lmwt-kiss.bundle#14]
Unless this is something I can easily change on my end (I've looked - doesn't look to be - but maybe I wasn't looking in the right place?) I'd really find it handy to have an Adult content on/off switch in preferences.  

The only reason I know adult content is even blocked is I went to load up a random anime that I'd never heard of today on the TV side of Primewire and I found that adult content is blocked! 

Is this one of those "It's a bigger deal than you think it is" kind of things? If it is - no worries at all! 

Thanks for the great app :D

**Comment by Twoure** [comment]
> The only reason I know adult content is even blocked is I went to load up a random anime that I'd never heard of today on the TV side of Primewire and I found that adult content is blocked!

I added the adult pop-up notification in my last merged pull-request https://github.com/piplongrun/lmwt-kiss.bundle/pull/11 because I though people would like to know why the content wasn't showing up.

> I'd really find it handy to have an Adult content on/off switch in preferences.

As for adding the preference, I looked into this awhile ago and found it to be dictated by a cookie.  At the time I wasn't as good at manipulating cookies within Plex's Framework as I am now, so will take a second look into this feature request.

> Is this one of those "It's a bigger deal than you think it is" kind of things? If it is - no worries at all!

If I recall correctly, the cookie is not global for all adult content, instead is different for each adult content page.  Will see once I have some time.

**Comment by Pyxie** [comment]
Thanks so much for the information! Like I said, if it’s too much trouble,
don’t worry about it.  You’ve built a great app already and that’s freakin
awesome :)



From: Twoure <notifications@github.com> <notifications@github.com>
Reply: piplongrun/lmwt-kiss.bundle
<reply@reply.github.com>
<reply@reply.github.com>
Date: August 28, 2016 at 11:46:29 AM
To: piplongrun/lmwt-kiss.bundle <lmwt-kiss.bundle@noreply.github.com>
<lmwt-kiss.bundle@noreply.github.com>
Cc: Pyxie <pyxieface@gmail.com> <pyxieface@gmail.com>, Author
<author@noreply.github.com> <author@noreply.github.com>
Subject:  Re: [piplongrun/lmwt-kiss.bundle] Feature Request - Add Adult
Content on/off to preferences (#14)

The only reason I know adult content is even blocked is I went to load up a
random anime that I'd never heard of today on the TV side of Primewire and
I found that adult content is blocked!

I added the adult pop-up notification in my last merged pull-request #11
<https://github.com/piplongrun/lmwt-kiss.bundle/pull/11> because I though
people would like to know why the content wasn't showing up.

I'd really find it handy to have an Adult content on/off switch in
preferences.

As for adding the preference, I looked into this awhile ago and found it to
be dictated by a cookie. At the time I wasn't as good at manipulating
cookies within Plex's Framework as I am now, so will take a second look
into this feature request.

Is this one of those "It's a bigger deal than you think it is" kind of
things? If it is - no worries at all!

If I recall correctly, the cookie is not global for all adult content,
instead is different for each adult content page. Will see once I have some
time.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/piplongrun/lmwt-kiss.bundle/issues/14#issuecomment-242984970>,
or mute the thread
